<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSelectionDialogBase</class>
 <widget class="QMainWindow" name="ImageSelectionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>1032</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DoRIAH Image Selection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/plugins/image_selection/bomb</normaloff>:/plugins/image_selection/bomb</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="styleSheet">
       <string notr="true">QSplitter::handle {
    background-color: lightGray;
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QFrame" name="webFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="WebView" name="webView" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="mapFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="mapControls">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Map</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="mapSelect">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>Choose the web map to serve as background.</string>
               </property>
               <property name="statusTip">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mapZoomIn">
               <property name="toolTip">
                <string>Zoom map in.</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/zoom-in</normaloff>:/plugins/image_selection/zoom-in</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mapZoomOut">
               <property name="toolTip">
                <string>Zoom map out.</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/zoom-out</normaloff>:/plugins/image_selection/zoom-out</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mapZoomNative">
               <property name="toolTip">
                <string>Zoom map to its native resolution.</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/zoom-native</normaloff>:/plugins/image_selection/zoom-native</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="mapZoomFit">
               <property name="toolTip">
                <string>Zoom map to visible aerials and AoI.</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/zoom-fit</normaloff>:/plugins/image_selection/zoom-fit</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="aerialControls">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Aerials</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="loadAerials">
               <property name="toolTip">
                <string>Load aerial image meta data.</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/films</normaloff>:/plugins/image_selection/films</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="loadAttackData">
               <property name="toolTip">
                <string>Load attack data.</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/calendar</normaloff>:/plugins/image_selection/calendar</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="loadAoi">
               <property name="toolTip">
                <string>Load an area of interest.</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/layer-shape-polygon</normaloff>:/plugins/image_selection/layer-shape-polygon</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="aerialsContrastEnhancement">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Enhance the contrast of aerials.</string>
               </property>
               <property name="text">
                <string>Contrast</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/contrast-stretch</normaloff>:/plugins/image_selection/contrast-stretch</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="popupMode">
                <enum>QToolButton::MenuButtonPopup</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_4">
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="aerialsGray">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Show missing aerials</string>
               </property>
               <property name="text">
                <string>Gray</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/flag-gray</normaloff>:/plugins/image_selection/flag-gray</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="popupMode">
                <enum>QToolButton::MenuButtonPopup</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="aerialsBlue">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Show not yet determined previews</string>
               </property>
               <property name="text">
                <string>Blue</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/flag-blue</normaloff>:/plugins/image_selection/flag-blue</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="popupMode">
                <enum>QToolButton::MenuButtonPopup</enum>
               </property>
               <property name="autoRaise">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="aerialsGreen">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Show previews</string>
               </property>
               <property name="text">
                <string>Green</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/flag-green</normaloff>:/plugins/image_selection/flag-green</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="popupMode">
                <enum>QToolButton::MenuButtonPopup</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="aerialsYellow">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Show images</string>
               </property>
               <property name="text">
                <string>Yellow</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/flag-yellow</normaloff>:/plugins/image_selection/flag-yellow</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="popupMode">
                <enum>QToolButton::MenuButtonPopup</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_2">
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="usageUnset">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Show aerials with unset usage.</string>
               </property>
               <property name="text">
                <string>unset</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/selection</normaloff>:/plugins/image_selection/selection</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="usageSelected">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Show selected aerials.</string>
               </property>
               <property name="text">
                <string>Selected</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/tick</normaloff>:/plugins/image_selection/tick</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="usageDiscarded">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="toolTip">
                <string>Show discarded aerials.</string>
               </property>
               <property name="text">
                <string>Discarded</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/cross</normaloff>:/plugins/image_selection/cross</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_3">
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="aerialsFreeze">
               <property name="toolTip">
                <string>Freeze the scene.</string>
               </property>
               <property name="text">
                <string>Freeze</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/freeze</normaloff>:/plugins/image_selection/freeze</iconset>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="exportSelectedImages">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/image-export</normaloff>:/plugins/image_selection/image-export</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="GEO">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/plugins/image_selection/GEO</normaloff>:/plugins/image_selection/GEO</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>34</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MapView" name="mapView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
          <property name="renderHints">
           <set>QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Pending map read request.</string>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="responseTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Duration of last map read request.</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Response time:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="responseElapsed">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Time stamp of last map read response.</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>ago</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mapResolution">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Maximum resolution of loaded map.</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Map resolution:</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nAerialsShown">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Showing   0 of   0 aerials</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WebView</class>
   <extends>QWidget</extends>
   <header>image_selection.web_view</header>
  </customwidget>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header>image_selection.map_view</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageSelectionDialogBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>866</x>
     <y>990</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
